geometry_msgs/TransformStamped mario_position
geometry_msgs/TransformStamped finish_line_position
geometry_msgs/TransformStamped[] items_position
